cpp/include/cudf/rolling.hpp

```diff
@@ -76,6 +76,28 @@ gdf_column* rolling_window(const gdf_column &input_col,
                            const gdf_size_type *min_periods_col,
                            const gdf_size_type *forward_window_col);
 
+/* --------------------------------------------------------------------------*
+ * @brief  Computes the rolling window function of the values in a column.
+ *
+ * This function is the same as the above rolling window interface except this is
```

Comment:
Can't refer to `the above`. I think API docs need to stand alone, but can refer to other APIs via @see.

Reply:
Changed that but not sure if this is the correct way to use @see though.

Reply:
No, not exactly.  :) See reference to @see below. However, I would prefer that the docs for each function stand alone. It's preferable to repeat wording in both places than to cross reference, unless it's referring to some huge amount of words.

```@see [name-list]
Make references to other entities or URLs. Names joined by either :: are intrepreted as a member of a class. Overloaded methods can be selected by incuding a list of argument types.``` http://www.agapow.net/programming/tools/doxygen-cheatsheet/